src/test/groovy/wolox/bootstrap/services/EnvironmentConfigurationServiceSpec.groovy

```diff
@@ -0,0 +1,27 @@
+package wolox.bootstrap.services
+
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.test.context.ContextConfiguration
+import spock.lang.Specification
+import wolox.bootstrap.configuration.AppConfig
+
+@ContextConfiguration(classes = [AppConfig.class])
+class EnvironmentConfigurationServiceSpec extends Specification {
+
+    @Autowired
+    EnvironmentConfigurationService service;
+
+    def "checking server.port property"() {
+        expect:
+        service.getProperty("server.port") == "8080"
+    }
+
+    def "checking invalid property"() {
+        when:
+        service.getProperty("non_existent");
+
+        then:
+        thrown(Exception)
```

Comment:
For this to be more readable you can add some indentation below `given:`, `when:` and `then:`
This applies to rest of test cases :)

Reply:
In Spock documentation examples, there is no indentation after a those labels, for instance:

http://spockframework.org/spock/docs/1.3/spock_primer.html#_conditions